### ISSUE TITLE
Cannot get files downloaded by browser after execution finished

### ISSUE THREAD

**Issue by MoSedky** [aerokube/selenoid#952]
So I have a scenario that needs to download files from the Chrome browser and then need to copy these files to the local machine before the container destroyed. Does it need to edit in docker file ? or already have some implemented flags for this?

**Comment by vania-pooh** [comment]
@MoSedky `/download/` API only works when browser is alive. Just copy everything before closing browser session.

**Comment by MoSedky** [comment]
Thanks for your quick response. Is it accepting wildcards? the downloaded file name is a timestamp

**Comment by vania-pooh** [comment]
@MoSedky no, this is a regular fileserver.

**Comment by MoSedky** [comment]
So I got a solution for this problem without dealing with Mount volume of Docker Containers. Following are the steps:
1. Get ${session_Id} of Selenium Webdriver. As I am using Python so it was (driver.session_id)
2. Download Proposed file using usual Selenium scripts 
3. Navigate to Chrome://downloads page and find latest downloaded file name using Javascript (("return document.querySelector('downloads-manager').shadowRoot.querySelector('#downloadsList downloads-item'). shadowRoot.querySelector('div#content  #file-link').text")) 
4. Use Wget. download to download the file with url:
http://selenoid_url/${session_Id}/file_name

And file downloaded successfully.

Thanks @vania-pooh for your support.
Closing Issue..
